awsenergylabelerlib/awsenergylabelerlib.py

```diff
@@ -70,21 +72,28 @@ def __init__(self,
                  account_thresholds=None,
                  security_hub_filter=None,
                  allow_list=None,
-                 deny_list=None):
+                 deny_list=None,
+                 allowed_regions=None,
+                 denied_regions=None):
         if all([allow_list, deny_list]):
             raise MutuallyExclusiveArguments('allow_list and deny_list are mutually exclusive.')
+        if all([allowed_regions, denied_regions]):
+            raise MutuallyExclusiveArguments('allowed_regions and denied_regions are mutually exclusive.')
         self._logger = logging.getLogger(f'{LOGGER_BASENAME}.{self.__class__.__name__}')
         self.landing_zone_thresholds = landing_zone_thresholds_schema.validate(landing_zone_thresholds) if \
             landing_zone_thresholds else LANDING_ZONE_THRESHOLDS
         self.account_thresholds = account_thresholds_schema.validate(account_thresholds) if account_thresholds \
             else ACCOUNT_THRESHOLDS
         self.security_hub_filter = security_hub_filter_schema.validate(security_hub_filter) if security_hub_filter \
             else SECURITY_HUB_FILTER
-        self.allow_list = self._validate_account_ids(allow_list) if allow_list else []
-        self.deny_list = self._validate_account_ids(deny_list) if deny_list else []
-        self.landing_zone_name = landing_zone_name
-        self._security_hub = SecurityHub(query_filter=self.security_hub_filter, region=region)
         self._landing_zone = LandingZone(landing_zone_name, self.landing_zone_thresholds, self.account_thresholds)
+        self.allow_list = self._validate_account_ids(allow_list, self._landing_zone.account_ids) if allow_list else []
+        self.deny_list = self._validate_account_ids(deny_list, self._landing_zone.account_ids) if deny_list else []
+        self.allowed_regions = self._validate_regions(allowed_regions) if allowed_regions else []
+        self.denied_regions = self._validate_regions(denied_regions) if denied_regions else []
+        self.landing_zone_name = landing_zone_name
+        self._security_hub = SecurityHub(query_filter=self.security_hub_filter, region=region,
```

Comment:
These arguments should also be aligned vertically since they are longer than a line.